### PR TITLE
[CELEBORN-98][IMPROVEMENT] Remove unreachable code block in master/work arguments

### PR DESCRIPTION
**Pull request by zwangsheng**
# [IMPROVEMENT] Remove never reach code block in master/worker arguments

### What changes were proposed in this pull request?
Host and port check will alway isn't null, due to default value.

Remove this un-reach code block to keep the code neat.

### Related issues.
https://issues.apache.org/jira/projects/CELEBORN/issues/CELEBORN-98?filter=allissues

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Review by AngersZhuuuu — APPROVED** [review]

**Comment by AngersZhuuuu** [comment]
Can you create a ISSUE and change title like `[CELEBORN-xxx][PR_TYPE] xxxx`

**Comment by zwangsheng** [comment]
> an you create a ISSUE and change title like `[CELEBORN-xxx][PR_TYPE] xxxx`

Thanks for the reminder, i will open issue and add some unit tests later.

File: master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.internal.Logging
+import org.apache.celeborn.common.util.Utils
+import org.scalatest.funsuite.AnyFunSuite
+
+class MasterArgumentsSuite extends AnyFunSuite with Logging {
+
+  test("Test build masterArguments") {
+    import org.apache.celeborn.common.CelebornConf.{MASTER_HOST, MASTER_PORT}
```
Comment (AngersZhuuuu) [inline master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala:28]: Move to import

Comment (zwangsheng) [reply master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala]: Done

File: master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import org.scalatest.funsuite.AnyFunSuite
+
+import org.apache.celeborn.common.CelebornConf
+import org.apache.celeborn.common.CelebornConf.{MASTER_HOST, MASTER_PORT}
+import org.apache.celeborn.common.internal.Logging
+import org.apache.celeborn.common.util.Utils
+
+class MasterArgumentsSuite extends AnyFunSuite with Logging {
+
+  test("Test build masterArguments") {
```
Comment (AngersZhuuuu) [inline master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala:29]: Lets also add `CELEBORN-98: xxx`

Comment (zwangsheng) [reply master/src/test/scala/org/apache/celeborn/service/deploy/master/MasterArgumentsSuite.scala]: OK, i'd like add more info to test case name

File: worker/src/test/scala/org/apache/celeborn/service/deploy/WorkerArgumentsSuite.scala [hunk 3]
```diff
@@ -27,7 +27,7 @@ import org.apache.celeborn.service.deploy.worker.WorkerArguments
 
 class WorkerArgumentsSuite extends AnyFunSuite with Logging {
 
-  test("Test build workerArguments") {
+  test("[CELEBORN-98] Test build workerArguments in different case\"") {
```
Comment (AngersZhuuuu) [inline worker/src/test/scala/org/apache/celeborn/service/deploy/WorkerArgumentsSuite.scala:30]: Remove `\"`

**Pull request merged**